src/App.vue

```diff
@@ -6,20 +6,24 @@ export default {
     return {
       demoTasks: [
         {
-          data: "demo task 1",
-          id: 0,
+          data: "Sample task 1",
+          id: 1646702392949,
+          status: true,
```

Comment:
variable name can be better
ex
`isComplete`
`isFinished`

Reply:
Sure